messaging/src/main/java/org/axonframework/common/caching/EhCacheAdapter.java

```diff
@@ -60,6 +61,21 @@ public boolean putIfAbsent(Object key, Object value) {
         return ehCache.putIfAbsent(new Element(key, value)) == null;
     }
 
+    @Override
+    public <T> T computeIfAbsent(Object key, Supplier<T> valueSupplier) {
```

Comment:
Looks like this method can be omitted to use the default implementation of the interface.